config/helm/chart/default/templates/Common/csi/daemonset.yaml

```
@@ -37,6 +37,9 @@ spec:
       labels:
         {{- include "dynatrace-operator.csiLabels" . | nindent 8 }}
     spec:
+      {{- if .Values.csi.tolerations }}
```

```suggestion
      {{- if (.Values.csi).tolerations }}
```

the csi section is quite new in the values.yaml, so just to be safe, this makes it null safe.
`--reuse-values` could break stuff without this (user doesn't have the csi section in there old values.yaml, and use `--reuse-values`, install will fail)